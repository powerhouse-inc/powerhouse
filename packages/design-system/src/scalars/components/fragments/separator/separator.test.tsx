import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";
import { Separator } from "./separator";

describe("Separator Component", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<Separator />);
    expect(asFragment()).toMatchSnapshot();
  });
});
