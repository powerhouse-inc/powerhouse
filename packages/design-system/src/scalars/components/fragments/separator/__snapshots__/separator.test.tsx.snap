// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Separator Component > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="shrink-0 bg-border h-[1px] w-full"
    data-orientation="horizontal"
    role="none"
  />
</DocumentFragment>
`;
